lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
What about

3. Run `flow version` (which is fast because it doesn't try to interact with or start the Flow server).

I think that would achieve the desired result (warning people if they cannot run Flow) without issuing extraneous warnings, and without damaging performance.

I'm not sure why a check about debugging would belong in this plugin. Flow is a typechecker only, and it seems strange to have  anything related to runtime involved here.